### PR TITLE
adding csslint into tests

### PR DESCRIPTION
**Pull request by fredex42**
adding in csslint dependency to tests, running csslint during test cycle

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -7,12 +7,14 @@ dependencies:
     #decrypt encrypted credentials - https://github.com/circleci/encrypted-files/blob/master/circle.yml
     - openssl aes-256-cbc -d -in credentials.cipher -k $KEY >> ~/.circlerc
     - source ~/.circlerc
+    - npm install -g csslint
```
Comment (akash1810) [inline circle.yml]: would be better to have this dependancy in a `package.json` and cache the `node_modules` directory to speed up builds

File: circle.yml [hunk 2]
```diff
@@ -7,12 +7,14 @@ dependencies:
...
   override:
     - sbt test:compile
 
 test:
   override:
     #- sbt test
+    - csslint --ignore=unique-headings app/assets/*.css
```
Comment (akash1810) [inline circle.yml]: would be better to have this as an `npm` command, such as `npm test`; makes it easier to chain tasks together and easier to run in DEV too.

File: circle.yml (hunk 1 above)
Comment (fredex42) [reply circle.yml]: done

**Comment by fredex42** [comment]
Ignore the test failure, it's because there is no css in this branch yet

File: circle.yml (hunk 1 above)
Comment (akash1810) [reply circle.yml]: think we can remove this line too.

**Comment by akash1810** [comment]
merging

**Pull request merged**